pkg/utils/pci_allocator.go

```diff
@@ -0,0 +1,83 @@
+package utils
+
+import (
+	"fmt"
+	"github.com/containernetworking/plugins/pkg/ns"
+	"os"
+	"path/filepath"
+)
+
+type PCIAllocation interface {
+	SaveAllocatedPCI(string, string) error
+	CleanAllocatedPCI(string) error
```

Comment:
nit: maybe `DeleteAllocatedPCI` ?

ill let you decide whatever has a nicer ring to it :)